app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ActiveRecord::Base
   # Include default devise modules.
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
-         :confirmable, :omniauthable
```

Comment:
Removing confirmable does mean no more tokens are generated, so if we add it back later, previous account will be invalid. However, overwriting the `confirmation_required?` method and then removing the overwrite will also make previous accounts invalid, unless someone goes through the backend to send existing users a confirmation email, which doesn't sound ideal. One thing we can do is configure devise so that users can access their account for a certain number of days without confirming their email, but then they have to confirm it after that point to continue use. Thoughts?